build.sbt

```diff
@@ -14,22 +14,24 @@ lazy val root = (project in file("."))
       "org.http4s"     %% "http4s-blaze-server" % Http4sVersion,
       "org.http4s"     %% "http4s-dsl"          % Http4sVersion,
       "org.specs2"     %% "specs2-core"         % Specs2Version % "test",
-      "ch.qos.logback"  %  "logback-classic"    % LogbackVersion
+      "ch.qos.logback" % "logback-classic"      % LogbackVersion
     ),
     addCompilerPlugin("org.typelevel" %% "kind-projector"     % "0.10.3"),
-    addCompilerPlugin("com.olegpy"     %% "better-monadic-for" % "0.3.1"),
+    addCompilerPlugin("com.olegpy"    %% "better-monadic-for" % "0.3.1"),
     mappings in Universal ++= directory(baseDirectory.value / "static"),
     buildInfoKeys := Seq[BuildInfoKey](name, version),
-    buildInfoPackage := "com.martinsnyder.chatserver"
+    buildInfoPackage := "com.martinsnyder.chatserver",
+    turbo := true
```

Comment:
I never got a chance to research the implications of this. Why isn't it the default?

Reply:
Not sure either. I guess because it is new, it won't be activated as default.
You should encounter faster starts of the app and tests within a sbt session.